cmd/auth.go

```diff
@@ -214,7 +217,31 @@ func runInteractiveLogin(ctx *cmdctx.CmdContext) error {
 	return flyctl.SaveConfig()
 }
 
-func runLogout(ctx *cmdctx.CmdContext) error {
+func runLogout(cc *cmdctx.CmdContext) error {
+	ctx := createCancellableContext()
+
+	captureError := func(err error) {
+		// ignore cancelled errors
+		if errors.Is(err, context.Canceled) {
+			return
+		}
+
+		flyerr.CaptureException(err,
+			flyerr.WithTag("feature", "logout"),
+		)
+	}
+
+	agentclient, err := agent.Establish(ctx, cc.Client.API())
+	if err != nil {
+		captureError(err)
+		return errors.Wrap(err, "can't establish agent")
+	}
+
+	if err := agentclient.Kill(ctx); err != nil {
+		captureError(err)
+		return errors.Wrap(err, "can't kill agent")
+	}
```

Comment:
`agent.Establish` would _start_ an agent if it isn't already running. There's some code to stop a running agent based on the pid file we write here: https://github.com/superfly/flyctl/blob/launch-redwood/pkg/agent/agent.go/#L127-L138

It's okay to ignore these errors since it's a cleanup process.

Reply:
Makes sense